hummingbot/strategy/pure_market_making/pure_market_making_config_map.py

```diff
@@ -79,6 +79,16 @@ def validate_price_floor_ceiling(value: str) -> Optional[str]:
         return "Value must be more than 0 or -1 to disable this feature."
 
 
+def validate_custom_api_max_price_age(value: int) -> Optional[str]:
```

Comment:
Looks like you can get related fields and based validation on that, and my understanding here is that
`value  >= pure_market_making_config_map.get("custom_api_update_interval").value` should be added as a condition.